KomaMRIBase/src/datatypes/phantom/motion/simplemotion/HeartBeat.jl

```diff
@@ -0,0 +1,49 @@
+@doc raw"""
+"""
+
+@with_kw struct HeartBeat{T<:Real} <: SimpleMotionType{T} 
+    ti::T
+    tf::T
```

Comment:
I think it is better to name these `start` and `end` and initialize them to zero to generate offsets.

- [ ] HearBeat
- [ ] Rotation
- [ ] Translation

Use `zero(T, 0.0)` instead of `typeof(ti)(0.0)`. 

- [ ] PeriodicHearBeat
- [ ] PeriodicRotation
- [ ] PeriodicTranslation
- [ ] HearBeat
- [ ] Rotation
- [ ] Translation